lib/jsdom/living/nodes/HTMLFrameElement-impl.js

```diff
@@ -1,16 +1,92 @@
 "use strict";
 const MIMEType = require("whatwg-mimetype");
+const whatwgEncoding = require("whatwg-encoding");
 const { parseURL, serializeURL } = require("whatwg-url");
+const sniffHTMLEncoding = require("html-encoding-sniffer");
+
 const { evaluateJavaScriptURL } = require("../window/navigation");
 const HTMLElementImpl = require("./HTMLElement-impl").implementation;
 const { applyDocumentFeatures } = require("../../browser/documentfeatures");
-const resourceLoader = require("../../browser/resource-loader");
 const { defineGetter, reflectURLAttribute } = require("../../utils");
 const { documentBaseURL } = require("../helpers/document-base-url");
 const { getAttributeValue } = require("../attributes");
 const idlUtils = require("../generated/utils");
 
-function loadFrame(frame) {
+function fireLoadEvent(document, frame, attaching) {
+  if (attaching) {
+    const ev = document.createEvent("HTMLEvents");
+
+    ev.initEvent("load", false, false);
+    frame.dispatchEvent(ev);
+
+    return;
+  }
+
+  const dummyPromise = new Promise(resolve => {
+    resolve();
+  });
+
+  function onLoad() {
+    const ev = document.createEvent("HTMLEvents");
+
+    ev.initEvent("load", false, false);
+    frame.dispatchEvent(ev);
+
+    return Promise.resolve();
+  }
+
+  document._queue.push(dummyPromise, onLoad);
+}
+
+function fetchFrame(serializedURL, frame, document, contentDoc) {
+  const { _resourceLoader: resourceLoader } = document;
```

Comment:
Avoid this syntax. Instead just do `const resourceLoader = document._resourceLoader`. Clearer and similar amount of typing. I think I saw this earlier too.

Reply:
IIRC I was doing that because of an eslint rule, but now it works. I will change them.